pjrpc/server/integration/aio_pika.py

```diff
@@ -80,17 +96,18 @@ async def _rpc_handle(self, message: aio_pika.abc.AbstractIncomingMessage) -> No
             if response_text is not None:
                 if reply_to is None:
                     logger.warning("property 'reply_to' is missing")
-                else:
-                    async with self._connection.channel() as channel:
-                        await channel.default_exchange.publish(
-                            aio_pika.Message(
-                                body=response_text.encode(),
-                                reply_to=reply_to,
-                                correlation_id=message.correlation_id,
-                                content_type=pjrpc.common.DEFAULT_CONTENT_TYPE,
-                            ),
-                            routing_key=reply_to,
-                        )
+                
+                async with self._connection.channel() as channel:
+                    exchange = self._exchange if self._exchange else channel.default_exchange
+                    await exchange.publish(
+                        aio_pika.Message(
+                            body=response_text.encode(),
+                            reply_to=reply_to,
+                            correlation_id=message.correlation_id,
+                            content_type=pjrpc.common.DEFAULT_CONTENT_TYPE,
+                        ),
+                        routing_key=self._tx_routing_key if self._tx_routing_key else reply_to,
```

Comment:
It seems to be necessary to check that `self._tx_routing_key` is not `None` or `reply_to` is not `None` otherwise `routing_key` will be `None` which is forbidden.

Reply:
Correct, thx! Fixed it by using default routing key "". This could be usesfull on cases where a fanout exchange is used as target.